docs/examples/rx/09_change_runtime_code/index.js

```diff
@@ -0,0 +1,135 @@
+const { combineLatest } = require('rxjs');
+const { concatMap, finalize, tap } = require('rxjs/operators');
+const { ApiRx } = require('@polkadot/api');
+const { WsProvider } = require('@polkadot/rpc-provider');
+
+// Initialise the provider to connect to the local node
+const wsProvider = new WsProvider('ws://127.0.0.1:9944');
+
+// Known account to use (available on dev chain with funds)
+// FIXME: How are we associating Alice's account with funding the
+// proposal and referendum creation
+const Alice = '5GoKvZWG5ZPYL1WUovuHW3zJBWBP5eT8CbqjdRY4Q6iMaDtZ';
+
+async function main () {
+  const api = await ApiRx.create(wsProvider);
+
+  const proposedNewRuntimeCode = new Uint8Array([4 << 2, 0xde, 0xad, 0xbe, 0xef]);
+  const voteYay = true;
+
+  // Create API. Wait until ready. Subscribe to API changes. Pass optional provider.
+  api
+    .pipe(
+      // `concatMap` subscribes to observables in order
+      concatMap((api) =>
+        combineLatest([
```

Comment:
This is an unworkable example, looks like callback hell :)

Reply:
@amaurymartiny I'm getting the following error and not sure how to overcome the callback hell-like beast that i've created. Any ideas? I'm also getting the following error:
```
/Users/me/code/src/polkadot-js/api/docs/examples/rx/09_change_runtime_code/node_modules/rxjs/internal/util/hostReportError.js:4
    setTimeout(function () { throw err; });
                             ^

TypeError: api.tx.consensus.setCode(...).pipe is not a function
    at MergeMapSubscriber.api.pipe.concatMap [as project] (/Users/me/code/src/polkadot-js/api/docs/examples/rx/09_change_runtime_code/index.js:43:14)
    at MergeMapSubscriber._tryNext (/Users/me/code/src/polkadot-js/api/docs/examples/rx/09_change_runtime_code/node_modules/rxjs/internal/operators/mergeMap.js:69:27)
    at MergeMapSubscriber._next (/Users/me/code/src/polkadot-js/api/docs/examples/rx/09_change_runtime_code/node_modules/rxjs/internal/operators/mergeMap.js:59:18)
    at MergeMapSubscriber.Subscriber.next (/Users/me/code/src/polkadot-js/api/docs/examples/rx/09_change_runtime_code/node_modules/rxjs/internal/Subscriber.js:67:18)
    at /Users/me/code/src/polkadot-js/api/docs/examples/rx/09_change_runtime_code/node_modules/rxjs/internal/util/subscribeToPromise.js:7:24
    at process._tickCallback (internal/process/next_tick.js:68:7)
```